sdk/core/Azure.Core/src/ChainingClassifier.cs

```diff
@@ -5,6 +5,17 @@
 
 namespace Azure.Core
 {
+    /// <summary>
+    /// Implements chaining of classifiers for the general case where the end-of-chain
+    /// <see cref="ResponseClassifier"/> is not a <see cref="CoreResponseClassifier"/>.
+    /// <see cref="CoreResponseClassifier"/> is preferred due to its enhanced performance
+    /// characteristics.
+    /// The classifier chain is a series of <see cref="ResponseClassificationHandler"/> classifiers
+    /// followed by the "end-of-chain" <see cref="ResponseClassifier"/>.  The handlers are
+    /// added to the chain via <see cref="RequestContext"/> or <see cref="RequestOptions"/>,
+    /// and applied starting with the most recently added handler, then applying status code
```

Comment:
Does the first step consider any handlers in order of most recently to least recently added or just the most recently added? I read this as the latter.

Reply:
Ah, yes, it is the former.  I will update this.